dask/dot.py

```diff
@@ -273,6 +273,10 @@ def dot_graph(dsk, filename="mydask", format=None, **kwargs):
 
 def graphviz_to_file(g, filename, format):
     fmts = [".png", ".pdf", ".dot", ".svg", ".jpeg", ".jpg"]
+
+    if format is None and filename is None:
+        format = "png"
+
     if format is None and any(filename.lower().endswith(fmt) for fmt in fmts):
```

Comment:
Should these two if statements be logically grouped with each other?

```python
if format is None and filename is None:
    pass
elif format is None and any(filename.lower().endswith(fmt) for fmt in fmts):
    # do the thing...
```

Reply:
Oh yes, this makes more sense. Updated :+1: